src/Symfony/Bundle/FrameworkBundle/Resources/config/cache_pools.xml

```diff
@@ -54,5 +54,7 @@
             <argument /> <!-- default lifetime -->
         </service>
 
+        <service id="cache.adapter.null" class="Symfony\Component\Cache\Adapter\NullAdapter" />
```

Comment:
This won't be used in Symfony I agree, but I think registering it here let you provide `cache.adapter.null` in framework configuration so you can disable cache. What do you think?